### ISSUE TITLE
Scaffolding Identity in .Net Core 6

### ISSUE THREAD

**Issue by JeffW12345** [dotnet/aspnetcore#40938]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The ['Scaffold Identity into a Razor project without existing authorization'](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-6.0&tabs=visual-studio#scaffold-identity-into-a-razor-project-without-existing-authorization) instructions for  don't work for .Net 6.0.

The following code was automatically added to my Program.cs file as a result of doing scaffolding for a Razor Pages app, using a context that did not previously exist:

builder.Services.AddDefaultIdentity<IdentityUser>(options => options.SignIn.RequireConfirmedAccount = true).
AddEntityFrameworkStores<ApplicationDbContext)<>();builder.Services.AddDbContext<ApplicationDbContext>(options => 
options.UseSqlServier(connectionString));

I got two error messages:

- IServiceCollection does not contain a definition for 'AddDefaultIdentity'.
- The name 'connectionString' does not exist in the current context.

What do you suggest, please?

Jeff

### Expected Behavior

I expected the scaffolding to add code that worked without any amendments. 

### Steps To Reproduce

Create a Razor Pages project and follow the instructions at https://docs.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-6.0&tabs=visual-studio#scaffold-identity-into-a-razor-project-without-existing-authorization

### Exceptions (if any)

N/A

### .NET Version

6.0

### Anything else?

No

**Comment by TanayParikh** [comment]
Hello @JeffW12345, thanks for reaching out. I believe this is currently blocked by https://github.com/dotnet/Scaffolding/issues/1666.

**Comment by JeffW12345** [comment]
> 

Hi Tanay

The link quoted on that page - https://github.com/aspnet/AspNetCore-ManualTests/issues/928 - does not work?

**Comment by TanayParikh** [comment]
https://github.com/aspnet/AspNetCore-ManualTests/issues/928 is an internal tracking issue. I cross-linked the issues internally but it's not publicly accessible at this time. We'll use https://github.com/dotnet/Scaffolding/issues/1666 and this new issue you've created to track the issue publicly 😄

**Comment by JeffW12345** [comment]
Thanks @[TanayParikh](https://github.com/TanayParikh). While this is being looked into, do you know of a workaround, please?

**Comment by adityamandaleeka** [comment]
Let's use https://github.com/dotnet/Scaffolding/issues/1666 to track this.
